chain/cosmos/src/data_source.rs

```diff
@@ -505,6 +509,8 @@ pub struct MappingMessageHandler {
 pub struct Source {
     #[serde(rename = "startBlock", default)]
     pub start_block: BlockNumber,
+    #[serde(rename = "endBlock")]
```

Comment:
https://serde.rs/container-attrs.html#rename_all  <- at this point might as well add this